Dockerfile

```diff
@@ -30,10 +28,8 @@ RUN if [ "${OVERWRITE_VERSION}" != "" ]; then \
 # build the api jar
 RUN mvn -T 8 clean package -Dquarkus.package.type=uber-jar -ntp
 
-
-
 ### Stage 3: build final application image
-FROM openjdk:11-jre-slim
+FROM openjdk:17.0.1-jdk-slim
```

Comment:
As the openjdk docker repository has been deprecated (see [here](https://hub.docker.com/_/openjdk)) in favor of eclipse temurin (see [here](https://github.com/docker-library/openjdk/issues/505)), can we adopt the eclipse temurin image instead (`eclipse-temurin:17-jre-alpine` or `eclipse-temurin:17-jre-focal`)?

Reply:
OH ok... then ya lets switch them.